dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/datavalue/DefaultDataValueService.java

```diff
@@ -143,7 +143,7 @@ public boolean addDataValue(DataValue dataValue) {
       if (config.isEnabled(CHANGELOG_AGGREGATE)) {
         DataValueAudit dataValueAudit =
             new DataValueAudit(
-                dataValue, dataValue.getValue(), dataValue.getStoredBy(), ChangeLogType.CREATE);
+                dataValue, dataValue.getValue(), dataValue.getStoredBy(), TrackerAuditType.CREATE);
```

Comment:
We are using `TrackerAuditType` here but are we sure `DataValue` is part of tracker?
To be honest I am not sure what is this `DataValue` but `DataValueService` is in core and I cannot see it living in tracker module.